arlo_server/all_routes/election/__element__.py

```diff
@@ -0,0 +1,91 @@
+from flask import jsonify, Response, request
+from jsonschema import validate
+
+from arlo_server import app, db
+from arlo_server.auth import require_audit_admin_for_organization
+from arlo_server.models import Election
+
+ELECTION_NAME_KEY = "electionName"
+ELECTION_ONLINE_KEY = "online"
+ELECTION_RANDOM_SEED_KEY = "randomSeed"
+ELECTION_RISK_LIMIT_KEY = "riskLimit"
+ELECTION_STATE_KEY = "state"
+
+PUT_ELECTION_REQUEST_SCHEMA = {
+    "type": "object",
+    "properties": {
+        ELECTION_NAME_KEY: {"type": "string"},
+        ELECTION_ONLINE_KEY: {"type": "boolean"},
+        ELECTION_RANDOM_SEED_KEY: {"type": "string"},
+        ELECTION_RISK_LIMIT_KEY: {"type": "integer"},
+        ELECTION_STATE_KEY: {"type": "string"},
+    },
+    "additionalProperties": False,
+}
+
+GET_ELECTION_RESPONSE_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "election": {
+            "type": "object",
+            "properties": {
+                ELECTION_NAME_KEY: {"type": "string"},
+                ELECTION_ONLINE_KEY: {"type": "boolean"},
+                ELECTION_RANDOM_SEED_KEY: {"type": "string"},
+                ELECTION_RISK_LIMIT_KEY: {"type": "integer"},
+                ELECTION_STATE_KEY: {"type": "string"},
+            },
+            "additionalProperties": False,
+        },
+    },
+    "additionalProperties": False,
+}
+
+
+@app.route("/election/<election_id>", methods=["GET"])
+def get_election(election_id: str) -> Response:
+    election = Election.query.filter_by(id=election_id).one()
+    require_audit_admin_for_organization(election.organization_id)
+
+    response_data = {
+        "election": {
```

Comment:
Why nest like this? So far, the endpoints I've seen don't use this pattern.

Reply:
The main reason to do this is if we ever want to include metadata in the response. We could add a key that's something like `_meta` or whatever, but doing it like this makes it much easier and less messy. It's the same reason I don't consider it a good idea to return JSON with an array as the root object: it makes it harder to extend later in a non-breaking way. I'll remove the wrapper object for now, but I think we still have a lot of work to do to come up with a good extensible API.

Reply:
Cool, that's what I figured. I agree with you 💯 %. Want to make a follow up task?

Reply:
Or I could be easily convinced to just start this pattern now and go back and fix existing endpoints later.

Reply:
We don't have to fix it now, and you and @MorganLove are likely to be the ones to own this going forward. So why don't you two think about it.